### PR TITLE
[UM-05] - add course method

### PR DESCRIPTION
**Pull request by Orujhimaru**
Screenshot 2024-05-28 at 3 36 04 PM
Screenshot 2024-05-28 at 3 36 04 PM

### CODE REVIEW

File: src/main/java/com/archisacadeny/course/CourseRepository.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import com.archisacadeny.config.DataBaseConnectorConfig;
+
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.MessageFormat;
+
+public class CourseRepository {
+
+    public static void addCourse(String course_name,String course_number,String course_instructor,String enrolled_students){
```
Comment (JamshidS) [inline src/main/java/com/archisacadeny/course/CourseRepository.java]: we will discuss how to insert data into a table in todays meeting. this is not the right way to do it.

**Review by JamshidS — CHANGES_REQUESTED** [review]

File: src/main/java/com/archisacadeny/course/CourseRepository.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+    public static void createCourseTable(){
+        try(Statement statement = DataBaseConnectorConfig.getConnection().createStatement())
+        {
+            // SQL query for creating persons table
+            String query = "CREATE TABLE IF NOT EXISTS courses(" +
+                    "course_id SERIAL PRIMARY KEY," + // id SERIAL ?
```
Comment (JamshidS) [inline src/main/java/com/archisacadeny/course/CourseRepository.java]: id
name
number
instructor_id

**Review by JamshidS — CHANGES_REQUESTED** [review]

**Review by HKalatas — COMMENTED** [review]
Course class ında olan field lar bunlar. SQL bunlara göre değiştirilmeli 

 private long id;
    private String courseName;
    private Instructor instructor;
    private long creditAmount;
    private long totalStudent;
    private List<Student> enrolledStudent;
    private String department;

File: src/main/java/com/archisacadeny/course/Course.java [hunk 3]
```diff
@@ -1,7 +1,52 @@
 package com.archisacadeny.course;
 
 import com.archisacadeny.config.DataBaseConfig;
+import com.archisacadeny.config.DataBaseConnectorConfig;
+
+import java.sql.SQLException;
+import java.sql.Statement;
 
 public class Course {
+//    private long id;
```
Comment (JamshidS) [inline src/main/java/com/archisacadeny/course/Course.java]: please remove all the comments

**Review by JamshidS — CHANGES_REQUESTED** [review]

**Review by JamshidS — APPROVED** [review]

**Pull request merged**